### DISCUSSION TITLE
Running Carla and Apollo using Carla build from source instead of docker

### DISCUSSION THREAD

**Discussion by adityadikshit07** [guardstrikelab/carla_apollo_bridge#89] [General]
Hi Team,

I am working on a project, where we have built carla from source and modified certain blueprints using Unreal editor, so basically our carla scenarios are running when we build carla inside unreal engine (using make launch).

In the bridge documentation, it is mentioned to run carla using docker.

Has anyone tried building carla using make launch and established communication with apollo ?
And if yes, are the steps different ?

**Comment by XiaoFei9704** [comment]
I've built carla from source, just follow the carla docs and it works well. Apollo or this bridge does not affect it.
